src/pages/using-spark/guides/content-style-guidelines.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Content Style Guidelines
+---
+
+# Content Style Guidelines
+
+UX Writers, designers and developers can use the **Content Style Guidelines** as a guide to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
+
+### Table Of Contents
+
+- [Capitalization](#capitalization)
+  - [Sentence Case](#sentence-case)
+  - [Start Case](#start-case)
+- [Quotation Marks](#quotation-marks)
+- [Accessibility Guidelines](#accessibility-guidelines)
+
+## Capitalization
+
+Use for visual presentation of pieces of text to make it easy for users to read.
+
+### Start Case
+
+Use this case style to capitalize the first letter of each word, regardless of length or part of speech. When using start case, avoid using end punctuation other than question marks.
+
+#### Usage
+
+Use for:
+
+- Headers that are descriptive or functional
+- All subheads regardless of whether the primary header is title or sentence case
+- All buttons labels (Primary, Secondary and Tertiary Buttons)
+- Menu items
+- Chart and table headers
+- Referring to the name of a page
+
+<h4 class=" docs-b-h4 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure ">
+  Examples:
+</h4>
+
+- Homeowners Insurance Info
+- How To Buy A Home
+- Lock Your Rate And Save
+- Check Your To Do List
+
+### Sentence Case
+
+Sentence case capitalizes the first word of the sentence and proper nouns.
+
+#### Usage
+
+Use for:
+
+- Emotional and Impactful headers
+- Special instances, where there may be two primary headers on a page
+- Body copy
+- Email subject lines and pre-headers
+- List items, use end punctuation only when the list item is a complete sentence
+
+<h4 class=" docs-b-h4 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure ">
+  Examples
+</h4>
+
+- We’re here to help!
+- If you want the lowest payment possible, you might want to look at a 30-year fixed.
+- Lock your rate and save.
+- Interest rates are on the rise.
+- Welcome to Rocket Mortgage.
+
+> As a general rule, use **Start case** for **Headers**, use either **Start** or **Sentence case** for drop down options and labels, depending on the context.
```

Comment:
Let's put this at the top,  right above "Start Case" header -- this gives the user a very fast summary of what they're about to read.